src/utils/style-formatter.js

```diff
@@ -42,18 +40,18 @@ export default {
     styles += formatColorProperty('color', style.fontColor, '#ffffff');
     styles += formatProperty('font-size', !isNaN(style.fontSize) ? `${style.fontSize}px` : '12px');
     styles += formatColorProperty('background-color', style.backgroundColor, primaryColor);
-    checkDisabled(style) && (styles += formatProperty('opacity', 0.4));
+    disabled && (styles += formatProperty('opacity', 0.4));
 
     if (style.background && style.background.isUsed) {
       let bgUrl = style.background.url.qStaticContentUrl.qUrl;
       bgUrl.replace(/^\.\.\//i, '/');
       bgUrl = bgUrl.replace(/"/g, '\\"');
       bgUrl = bgUrl.replace(/'/g, "\\'");
       styles += formatProperty('background-image', `url('${bgUrl}')`);
-      styles += formatProperty('background-size', backgroundSize[style.background.size] || backgroundSize.auto);
+      styles += formatProperty('background-size', backgroundSize[style.background.size]);
```

Comment:
I've removed the default values, so we need to keep this or put the default values back.

Reply:
ah, I got confused then - I'll put them back. :)